src/app/history/(components)/OrderTable/TableRowOrder.tsx

```diff
@@ -53,23 +82,16 @@ export function TableRowOrder({ order }: { order: StopLossOrderType }) {
             "Error loading token"
           )}
         </TableCell>
-        <TableCell>{capitalize(order?.status as string)}</TableCell>
-        <TableCell>
-          <button
-            type="button"
-            className="flex items-center"
-            onClick={CancelOrder}
-            disabled={disabled}
-          >
-            <TrashIcon
-              className={cn(
-                "size-5",
-                disabled
-                  ? "text-foreground-primary/50"
-                  : "text-destructive hover:destructive/80"
-              )}
-            />
-          </button>
+        <TableCell>{StatusBadge(order?.status)}</TableCell>
+        <TableCell 
+          onClick={(e) => e.stopPropagation()}
```

Comment:
what were we propagating here? 👀

Reply:
added a comment

 // Stop onClick from table row, which is redirecting to order details page